packages/core/src/components/Dialog/index.ts

```diff
@@ -1,4 +1,5 @@
 export * from "./Title";
+export * from "./Backdrop";
```

Comment:
This means that `HvDialogBackdrop` is now a new component (exported at the root level) that's part of the API - We don't want that!

Reply:
One more try to go...